### DISCUSSION TITLE
standaard-2-de-onderwijsleeromgeving

### DISCUSSION THREAD

**Discussion by fjkresin** [fdnd/docs.fdnd.nl#278] [Reflectie]
"Tijdens sprints werken studenten aan leertaken. In onderstaande afbeelding is ingezoomd op semester 1, sommige sprintnamen zijn inmiddels aangepast." - Dan zou ik een nieuw plaatje maken.

**Comment by ju5tu5** [comment]
Ja je hebt helemaal gelijk.. ik vraag het Charley even..
